packages/back-end/src/tracing.datadog.ts

```diff
@@ -6,15 +6,63 @@ tracer.init({
   logInjection: true,
 });
 
+const COLLECTION_INTERVAL = 15;
```

Comment:
suggestion: Can we add the unit for this? is it seconds or milliseconds? For example `COLLECTION_INTERVAL_MS = `